examples/multimodal/clip.ts

```diff
@@ -1,5 +1,16 @@
+import * as fs from "fs";
```

Comment:
use { defaultFS } from `@llamaindex/env`

Reply:
we are aiming to support multiple js runtime, so it's necessary to wrap system API into another package